index.rst

```diff
@@ -360,6 +390,15 @@ fourth-level element and its children.
    What is the consequence for missing a milestone? It has no direct earned
    value impact.
 
+.. note::
+
+   No, level 1 milestones are exposed to NSF. NSF picked milestones they
+   wanted to see, making them level 1. ProjMgr should weight in on level 2
+   because they involve other subsystems. For level 3, if a milestone is
+   related to say team x blocking team y, the tcams of these two teams
+   should decide.
+
```

Comment:
Updated text to reflect this.

It's still unclear what the consequence of missing an L1 milestone is. I mean, it's clearly _bad_, in the sense that it gets a hard stare from NSF, but does it trigger any action equivalent to writing a variance narrative?